thirdparty/ovr_openxr_mobile_sdk/OpenXR/Include/openxr/extx1_event_channel.h

```diff
@@ -1,103 +0,0 @@
-#ifndef EXTX1_EVENT_CHANNEL_H_
```

Comment:
Removing Meta's OpenXR headers isn't totally related to removing Meta's loader, but I've wanted to remove these for a long time, so I support it!

(I had intended to pair the removal with the addition of a build argument to include them from somewhere on the developer's system, so we can still use them for Meta's OpenXR extensions that aren't in the spec, but I can add that in a future PR :-))